RELEASE-NOTES.md

```diff
@@ -24,6 +24,10 @@
 ## Deprecations
 
 ## New additions
+* Added support for `native_app.package.post_deploy` scripts in project definition file.
+  * These scripts will execute whenever a Native App Package is created or updated.
+  * Currently only supports SQL scripts: `post_deploy: [{sql_script: script.sql}]`
+  * Added deprecation warning for `native_app.package.scripts`
```

Comment:
Should this bullet move to be under the Deprecations section?

Reply:
moved